### PR TITLE
@craigspaeth => Mobile Panel video, allow empty links

### PR DESCRIPTION
**Pull request by eessex**
- Use `cover_image_url` instead of `assets` for video cover images
- Adds mobile styles for Panel video (cover image has play button, plays on click)
- Moves video caret/controls into reusable component
- Allows for unit.link to be empty

I added a tracking event for the video play with `action: "Click"` -- is there a way to make this accommodate both click and mouseenter, or should we make separate events?

panel-video

### CODE REVIEW

File: src/Components/Publishing/Display/DisplayPanel.tsx [hunk 1]
```diff
@@ -47,70 +60,89 @@ export class DisplayPanel extends React.Component<DisplayPanelProps, null> {
...
+    const { link } = this.props.unit
     e.preventDefault()
-    window.open(this.props.unit.link.url, '_blank')
+    if (link) {
+      window.open(link.url, '_blank')
+    }
   }
 
-  onMouseEnter = () => {
+  @track(once((props) => ({
```
Comment (craigspaeth) [inline src/Components/Publishing/Display/DisplayPanel.tsx]: `once` is probably not necessary here as I believe we'll actually want to track multiple plays.

File: src/Components/Publishing/Display/DisplayPanel.tsx [hunk 2]
```diff
@@ -47,70 +60,89 @@ export class DisplayPanel extends React.Component<DisplayPanelProps, null> {
...
   renderVideo = (url) => {
+    const { isMobile } = this.props
     return (
-      <VideoContainer>
-        <VideoCover />
+      <VideoContainer onClick={isMobile && this.playVideo}>
+        {!this.state.isPlaying &&
+          <VideoCover>
+            {isMobile &&
+              <VideoControls mini />
```
Comment (craigspaeth) [inline src/Components/Publishing/Display/DisplayPanel.tsx]: 👍

**Comment by craigspaeth** [comment]
We should make the mouseenter and click tracking separate—we can always join them into the same thing via SQL/Looker later but conceptually they are different things. Minus the comment about `once` the code LGTM! Feel free to merge when you address that.

**Comment by eessex** [comment]
@craigspaeth I separated the events for mobile and desktop plays, they are distinguished by the labelling of their actions: "Click" and "MouseEnter"

**Pull request merged**